BUILD.bazel

```diff
@@ -0,0 +1,166 @@
+# Allow any ts_library rules in this workspace to reference the config
+exports_files(["tsconfig.json"], visibility = ["//:__subpackages__"])
+
+package(default_visibility = ["//visibility:public"])
+
+load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary", "nodejs_test")
+load("@npm_bazel_typescript//:index.bzl", "ts_project", "ts_config")
+load("@npm_bazel_labs//:index.bzl", "protobufjs_ts_library")
+
+proto_library(
+    name = "google_protobuf_protos",
+    srcs = glob(["node_modules/google-gax/build/protos/google/protobuf/*.proto"]) +
+    [
+        "node_modules/google-gax/build/protos/google/protobuf/compiler/plugin.proto",
+    ],
+    strip_import_prefix = "node_modules/google-gax/build/protos",
+)
+
+proto_library(
+    name = "google_rpc_protos",
+    deps = [
+        ":google_protobuf_protos",
+    ],
+    srcs = glob(["node_modules/google-gax/build/protos/google/rpc/*.proto"]),
+    strip_import_prefix = "node_modules/google-gax/build/protos/",
+)
+
+proto_library(
+    name = "google_api_protos",
+    deps = [
+        ":google_protobuf_protos",
+    ],
+    srcs = glob(["node_modules/google-gax/build/protos/google/api/*.proto"]),
+    strip_import_prefix = "node_modules/google-gax/build/protos",
+)
+
+proto_library(
+    name = "google_longrunning_protos",
+    deps = [
+        ":google_protobuf_protos",
+        ":google_api_protos",
+        ":google_rpc_protos",
+    ],
+    srcs = ["node_modules/google-gax/build/protos/google/longrunning/operations.proto"],
+    strip_import_prefix = "node_modules/google-gax/build/protos",
+)
+
+proto_library(
+    name = "service_config_proto",
+    deps = [
+        ":google_protobuf_protos",
+        ":google_rpc_protos",
+    ],
+    srcs = ["protos/service_config.proto"],
+    strip_import_prefix = "protos/",
+)
+
+protobufjs_ts_library(
+    name = "protos",
+    deps = [
+        ":google_api_protos",
+        ":google_longrunning_protos",
+        ":service_config_proto",
+    ]
+)
+
+filegroup(
+    name = "templates",
+    srcs = glob(["templates/**/*.*", "templates/**/.*"]),
+)
+
+filegroup(
+    name = "test-fixtures",
+    srcs = glob(["test-fixtures/**/*.*"]),
+)
+
+filegroup(
+    name = "baselines",
+    srcs = glob(["baselines/**/*.*", "baselines/**/.*"]),
+)
+
+npm_runtime_dependencies = [
+    "@npm//file-system",
+    "@npm//fs-extra",
+    "@npm//get-stdin",
+    "@npm//google-gax",
+    "@npm//js-yaml",
+    "@npm//module-alias",
+    "@npm//nunjucks",
+    "@npm//object-hash",
+    "@npm//protobufjs",
+    "@npm//yargs",
+]
+
+npm_test_dependencies = npm_runtime_dependencies + [
+    "@npm//assert-rejects",
+    "@npm//c8",
+    "@npm//codecov",
```

Comment:
Do you specifically call into codecov, or rely on the GitHub Action like we do in libs?

Reply:
I'm not calling it, this should be removed from here.  Actually, I have no idea how to call `codecov` and `c8` from bazel but I'll figure out :)